src/07-environment.md

```diff
@@ -46,31 +46,31 @@ that it also has an instance of `Functor`, `Applicative`, `Monad` and `MonadTran
 
 As we can see in the definition, `ReaderT` is *a newtype* over a function that takes
 some value of type `r`, and returns a value of type `m a`. The `r` usually
-represents the environment we want to share between different functions that we want to
-compose together, and the `m a` represents the underlying result that we return.
+represents the environment we want to share between the nested functions,
```

Comment:
I'm not sure that nested is the right term here, so I'll default to the original phrasing with a slight tweak. 
```suggestion
represents the environment we want to share between functions that we want to compose,
```